config/environments/development.rb

```diff
@@ -54,4 +54,11 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  
+  config.middleware.insert_before 0, Rack::Cors do
+    allow do
+      origins "https://localhost:3001", "http://localhost:3001"
+      resource '/api/v1/*', headers: :any, methods: :any, vary: "Origin", credentials: true
+    end
+  end
```

Comment:
It looks like the configuration for other environments is missing.

What about staging/production?

I'd suggest moving this configuration to `config/application.rb`. In order to share this config along other envs.

Also, I know the permited origins would change depending on the env
WDYT about setting the permitted origins via env-vars?

It'd be something like this:
```ruby
config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins "https://ENV["YOUR_PERMITTED_ORIGIN_URL"]", "http://YOUR_PERMITTED_ORIGIN_URL"
      resource '/api/v1/*', headers: :any, methods: :any, vary: "Origin", credentials: true
    end
  end
```

worth thinking for a better naming (it's always a thing huh).
It could be `TRIBO_REACT_NATIVE_ORIGIN` or something like that

Reply:
Ready!